src/draw.c

```diff
@@ -618,6 +644,38 @@ static void render_content(cairo_t *c, struct colored_layout *cl, int width)
                 cairo_fill(c);
         }
 
+        // progress bar positioning
+        if (have_progress_bar(cl->n)){
+                int progress = cl->n->progress;
```

Comment:
```suggestion
                int progress = MIN(cl->n->progress, 100);
```
To catch any misuse of the progress value.

Reply:
I fixed it in notification.c to prevent code duplication